includes/Checker/Checks/Plugin_Repo/Plugin_Header_Fields_Check.php

```diff
@@ -74,6 +69,16 @@ public function run( Check_Result $result ) {
 			'RequiresPlugins' => 'Requires Plugins',
 		);
 
+		$restricted_labels = array(
+			'BitbucketPluginURI' => 'Bitbucket Plugin URI',
+			'GistPluginURI'      => 'Gist Plugin URI',
+			'GiteaPluginURI'     => 'Gitea Plugin URI',
+			'GitHubPluginURI'    => 'GitHub Plugin URI',
+			'GitLabPluginURI'    => 'GitLab Plugin URI',
+		);
```

Comment:
Do we want to block just these fields, or should we just block anything that is not supported by `get_plugin_data`?

Reply:
We should not block like that. Currently this list contains fields that are used for updater which we don't allow. Other non-core fields are frequently seen like `Elementor tested up to`, `WC tested up to`, etc which should be allowed.